framework/test/jest/unit/specs/modules/chain/synchronizer/fast_chain_switching_mechanism/fast_chain_switching_mechanism.spec.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	FastChainSwitchingMechanism,
+} = require('../../../../../../../../src/modules/chain/synchronizer/fast_chain_switching_mechanism');
+const {
+	Block: blockFixture,
+} = require('../../../../../../../mocha/fixtures/blocks');
+
+describe('fast_chain_switching_mechanism', () => {
+	afterEach(async () => {
+		jest.clearAllMocks();
+	});
+
+	describe('FastChainSwitchingMechanism', () => {
+		const activeDelegates = 101;
```

Comment:
Same as the previous comments, this number should be retrieved from constants/config.

Reply:
Same as above.

Reply:
if it's a constant, it should be at least written in upper case

Reply:
Its passed as function argument and argument name is exactly `activeDelegates`